bench/src/metrics.rs

```diff
@@ -44,10 +47,20 @@ impl Metric {
                 self.total_sent_time.as_millis() as f64 / self.txs_sent as f64;
         }
 
+        if self.total_gross_send_time_ms > 0.01 {
+            self.send_tps =
+                1000.0 * self.txs_sent as f64 / self.total_gross_send_time_ms;
```

Comment:
please use () to make this more readable and obvious

Reply:
very good point! will do